azerothcore/azerothcore-wotlk
# fix(DB/creature): Improve AQ40 Bug Tunnel

## Changes Proposed:
-  Link Vekniss Drone spawns that are contained in a formation to Fankriss
-  Link Vekniss Soldier spawns to Fankriss
-  Creates linked formations to Vekniss Drones packs
-  Adjusts wander_distance to match a 20-30 minute sniff
-  Adjusts respawn timers for Drones and Soldiers
-  Adds SAI to Drones to force despawn after 5s

## Issues Addressed:

- Closes unopened issue

## SOURCE:

- Retail for wander_distance, formation and linking
- Vmangos for respawn timers.

## Tests Performed:

- Tested ingame


## How to Test the Changes:


1. `.go xyz -8205.5 1679 -30 531 6.12`
2. pull the drones, see if most are pulled in packs of 5. Note: there are several unlinked spawns, which is intentional
3. kill some Drones, see if they despawn in 5s and respawn in 25s
4. kill Fankriss, see if **most** Drones and all Soldiers do not respawn

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
